src/sage/rings/number_field/number_field.py

```diff
@@ -1880,17 +1887,17 @@ def _convert_non_number_field_element(self, x):
         will convert to the number field, e.g., this one in
         characteristic 7::
 
-            sage: f = GF(7)['y']([1,2,3]); f                                            # optional - sage.rings.finite_rings
+            sage: f = GF(7)['y']([1,2,3]); f                                            # needs sage.rings.finite_rings
             3*y^2 + 2*y + 1
```

Comment:
This is not needed anymore for prime finite fields?

Reply:
yes, for small enough prime finite fields